WTWSMS/common/landed_titles/WtWSMS_pagan_religious_heads.txt

```diff
@@ -632,7 +632,7 @@ d_jewish = {
 	dynasty_title_names = no 	# Will not be named "Seljuk", etc.
 }
 
-d_melchisedechianism = {
+k_melchisedechianism = {
```

Comment:
You make an argument on this, you need to prove it. I'm not at all convinced it has to be a duchy, I want to know why it can't be a duchy called a kingdom?